MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: post
+title: Grouping in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the grouping support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Grouping editors in .NET MAUI DataForm (SfDataForm)
+
+The .NET MAUI DataForm support to group the editors which are relevant to each other. You can expand or collapse the group by tapping the group item.
+
+Grouping can be achieved by defining [Display] attribute or by handling the `GenerateDataFormItem` event.
+
+#### Using attributes
+
+{% tabs %}
+{% highlight C# %}
+
+public class ContactsInfo
+{
+    [Display(GroupName = "Name")]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string MiddleName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string LastName { get; set; }    
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName" || e.DataFormItem.FieldName == "MiddleName" || e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.GroupName = "Name";
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing editors order in the group
```

Comment:
Can we split this into 2 topic

#### Using attribute

**DisplayAttribute**

**DataFormDisplayOptionsAttribute**
Mention about RowOrder and ItemsOrderInRow

#### Using event

Reply:
changed content